server/models/User.js

```diff
@@ -7,6 +7,11 @@ const userSchema = new mongoose.Schema({
     required: true,
     unique: true,
   },
+  picture: {
+    type: new mongoose.Schema({
+      url: { type: String, required: false },
+    }),
+  },
```

Comment:
What's happening here @eng-dallavecchia? Is `picture` referencing a string in the end? If so, just make it directly related to `{ type: String, required: false }`